mopro-web/docs/circom/ios.md

```diff
@@ -0,0 +1,184 @@
+# iOS
+
+## Getting started with a new project
+
+1. Install `mopro` cli. See [Gettin Started](../getting-started#install-dependencies).
+2. Create a new project
+
+```sh
+mopro init --platforms ios \
+cd mopro-example-app
+```
+
+3. Prepare circuits
+
+```sh
+mopro prepare
+```
+
+4. Build the project
+
+```sh
+mopro build --platforms ios
+```
+
+5. Open the project in xcode
+
+```sh
+open ios/ExampleApp/ExampleApp.xcworkspace
+```
+
+## Getting started with exported bindings
+
+1. Install `mopro` cli. See [Gettin Started](../getting-started#install-dependencies).
+2. Prepare circuits
+
+```sh
+mopro prepare
+```
+
+3. Build the project
+
+```sh
+mopro build --platforms ios
+```
+
+4. Export bindings
+
+```sh
+mopro export-bindings --platforms ios --destination out
+```
+
+5. Create an `xcframework` with `xcodebuild`
```

Comment:
I thought we scripetd this part?

Reply:
we did this in building dylib
but I think we can include this in `mopro build` script too!